### ISSUE TITLE
Dependencies problems

### ISSUE THREAD

**Issue by rodolfobarretoweb** [l-urence/react-native-autocomplete-input#13]
Hello, i try to install but i get some erros of dependence. I solution this by install all packages bellow:

babel-plugin-transform-es2015-arrow-functions
babel-plugin-transform-es2015-modules-commonjs
babel-plugin-transform-object-rest-spread
babel-plugin-transform-react-jsx

**Comment by l-urence** [comment]
I can install the the autocomplete component from npm and github without any errors. Did you try to install it into a clean RN project?

**Comment by rodolfobarretoweb** [comment]
Hello. No, i have some modules installed

**Comment by l-urence** [comment]
I removed the Babel configuration from package.json. Can pleases try to install the component from GitHub `npm install --save l-urence/react-native-autocomplete-input`.

**Comment by patrickshaughnessy** [comment]
I had a similar problem. Set up the autocomplete component no problem on the simulator, but when I tried to run it on a real device I kept getting JS syntax error:

`SyntaxError: Unexpected keyword 'const'. Const declarations are not supported in strict mode.`

If I understood the docs correctly, the xcode simulator has access to Chrome's v8 engine during debugging, but only the core engine on the device. Not sure how that affects your plugin exactly, but that's my guess.

Anyways, I reinstalled the autocomplete package from github as you suggested and it works now.

Thanks!
